addons/slideshow/CfgVehicles.hpp

```diff
@@ -0,0 +1,130 @@
+class CfgVehicles {
+    class ACE_Module;
+    class GVAR(module): ACE_Module {
+        author = ECSTRING(common,ACETeam);
+        category = "ACE_missionModules";
+        displayName = CSTRING(DisplayName);
+        function = QFUNC(moduleInit);
+        scope = 2;
+        isGlobal = 0; // Server only
+        isTriggerActivated = 0;
+        isDisposable = 0;
+        icon = QUOTE(PATHTOF(UI\Icon_Module_Slideshow_ca.paa));
+        class Arguments {
+            class Objects {
+                displayName = CSTRING(Objects_DisplayName);
+                description = CSTRING(Objects_Description);
+                typeName = "STRING";
+                defaultValue = "";
+            };
+            class Controllers {
+                displayName = CSTRING(Controllers_DisplayName);
+                description = CSTRING(Controllers_Description);
+                typeName = "STRING";
+                defaultValue = "";
+            };
+            class Images {
+                displayName = CSTRING(Images_DisplayName);
+                description = CSTRING(Images_Description);
+                typeName = "STRING";
+                defaultValue = "";
+            };
+            class Names {
+                displayName = CSTRING(Names_DisplayName);
+                description = CSTRING(Names_Description);
+                typeName = "STRING";
+                defaultValue = "";
+            };
+            class Duration {
+                displayName = CSTRING(Duration_DisplayName);
+                description = CSTRING(Duration_Description);
+                typeName = "NUMBER";
+                defaultValue = 0;
+            };
+        };
+        class ModuleDescription {
+            description = CSTRING(Description);
+
+            // Every object in this must be initialized with GVAR(Slides) Action applied, as well as 2 of below macros
+            sync[] = {CLASSNAMES_OBJECTS, CLASSNAMES_BOTH, CLASSNAMES_CONTROLLERS};
+
+            #define MACRO_SYNC \
+                optional = 1; \
+                duplicate = 1;
+            #define OBJECT description = "Object";
+            #define BOTH description = "Object AND Controller";
+            #define CONTROLLER description = "Controller";
```

Comment:
They are: https://github.com/jonpas/ACE3/blob/slideshow/addons/slideshow/CfgVehicles.hpp#L58-L70

Maybe a bit hard to notice. This is just for module "info" tab though, if a way to make it all dynamic (again ACE interaction enabling is required for the objects) then that wouldn't be needed at all.